indico/queries/model_groups/model_groups.py

```diff
@@ -236,3 +236,104 @@ def requests(self):
         yield _ModelGroupPredict(
             model_id=self.model_id, data=self.data, predict_options=self.predict_options
         )
+
+
+class UpdateModelGroupSettings(GraphQLRequest):
+    """
+    Updates an existing model group component in the platform.
+
+    Args:
+        model_group_id (int): the id of the model group to update settings
+        model_training_options (dict, optional): model training options to use when training model. Defaults to None
+        predict_options (dict, optional): predict options to use on model. Defaults to None
+        domain (str, optional): feature domain to use.
+            Valid values include STANDARD, STANDARD_V2, TOPICS, SENTIMENT, FINANCE, EMOTION, ENSEMBLE, FASTTEXT, UNSUPERVISEDSENTIMENT, IMAGE_V2, IMAGE_V3, IMAGE_V4, IMAGE_ENSEMBLE
+            Defaults to None
+        finetune (bool, optional): flag to use finetune or not in model training. Defaults to None
+        interlabeler_resolution (str, optional): denotes label resolution strategy.
+            Valid values include MAJORITY_VOTE_WITH_TIES, MAJORITY_VOTE_WITHOUT_TIES, UNANIMOUS, ALL
+            Defaults to None
+        make_predictions (bool, optional): flag to make predictions or not. Defaults to None
+        roc_auc_averaging (str, optional): roc auc averaging.
+            Valid values include SIMPLE, WEIGHTED
+            Defaults to None
+        sampling_strategy (str, optional): denote sampling strategy to use.
+            Valid values include NO_SAMPLING, RANDOM_OVERSAMPLE
+            Defaults to None
+        task_type (str, optional): denotes task type.
+            Valid values include CLASSIFICATION, CLASSIFICATION_MULTIPLE, RATIONALIZED_CLASSIFICATION, REGRESSION, ANNOTATION, OBJECT_DETECTION, FORM_EXTRACTION, CLASSIFICATION_UNBUNDLING
+            Defaults to None
+        test_split (float, optional): denotes test split. Defaults to None
+        word_predictor_strength (str, optional): denotes word predictor strength
+            Valid values include STRONG, MODERATE, WEAK
+            Defaults to None
+    """
+
+    query = """
+        mutation updateModelGroup($modelGroupId: Int!, $modelTrainingOptions: JSONString) {
```

Comment:
How come with only have 2 args for this query if there's like 12 variables?

Reply:
same question here - think you're missing some mutation args